cdhweb/people/templates/people/profile_page.html

```diff
@@ -20,8 +20,8 @@ <h1>{{ page.title }}</h1>
 <div class="links">
     {# NOTE equiv to contributors in project #}
     <ul>
-    {% for resource in page.person.personresource_set.all %}
-        <li><a href="{{ resource.url }}">{{ resource.resource_type.name }}</a></li>
+    {% for resource in page.person.personrelatedlink_set.all %}
+        <li><a href="{{ resource.url }}">{{ resource.type.name }}</a></li>
```

Comment:
because you set a `related_name` for this field, you should just be able to do `person.related_links.all`, which is a little nicer

Reply:
I'd also change it to `{% for link in ...%}`, `link.url`, etc

Reply:
addressed by 9b415a9